### PR TITLE
fix: [ANDROAPP-2799] Adds complete/deactivated option to tei dashboard menu

### PR DESCRIPTION
**Pull request by ferdyrod**
**JIRA ISSUE**: [ANDROAPP-2799](https://jira.dhis2.org/browse/ANDROAPP-2799)

### CODE REVIEW

File: app/src/main/java/org/dhis2/usescases/teiDashboard/TeiDashboardPresenter.java [hunk 1]
```diff
@@ -266,6 +267,28 @@ public void handleShowHideFilters(boolean showFilters) {
...
+                .observeOn(schedulerProvider.ui())
+                .subscribe(updated -> {
+                    if (updated) {
+                        view.updateStatus();
+                    } else {
+                        view.displayMessage("There was an error updating the status");
+                    }
+                }, Timber::e)
+        );
+    }
```
Comment (JaimeToca) [inline app/src/main/java/org/dhis2/usescases/teiDashboard/TeiDashboardPresenter.java]: unit tests :D ?

**Comment by JaimeToca** [comment]
@ferdyrod everything okay. just added a comment regarding unit test

File: app/src/main/java/org/dhis2/usescases/teiDashboard/TeiDashboardPresenter.java (hunk 1 above)
Comment (ferdyrod) [reply app/src/main/java/org/dhis2/usescases/teiDashboard/TeiDashboardPresenter.java]: done!

**Comment by ferdyrod** [comment]
@JaimeToca @Balcan done with your comments!

**Pull request merged**